### PR TITLE
Global / console cleanup

### PR DESCRIPTION
**Pull request by NotEternal**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/MultiCurrencyWallet/blob/master/docs/CONTRIBUTING.md) guide
- [ ] Good naming (as clear and simple as possible)
- [ ] Correct behavior if external API endpoints are down, return 404, 504 (or no answer), 401 errors (ddos simulation)
- [ ] I tested desktop/mobile resolution
- [x] I tested light/dark theme
- [ ] I tested different languages
- [ ] I checked the functionality once again (**AFFECT MONEY**)
- [x] I checked the work on the Testnet
- [x] I checked the work on the Mainnet
- [ ] I checked the work in the plugin
- [ ] I checked the **PR** once again

## Tests

Please start auto tests as follows:

- add a label <ins>swap test</ins> to start swap tests
- add a label <ins>withdraw test</ins> to start withdraw tests

You can skip these tests if you completely sure that your changes aren't related to this functional

### Original issue
#3511
#4044

### CODE REVIEW

**Comment by developer24-exe** [comment]
builded https://testnet4809.swaponline.site ;

**Comment by developer24-exe** [comment]
commit e79e2dc96 rebuilded https://testnet4809.swaponline.site https://mainnet4809.swaponline.site

**Comment by developer24-exe** [comment]
commit a76d984c3 rebuilded https://testnet4809.swaponline.site https://mainnet4809.swaponline.site

**Comment by developer24-exe** [comment]
commit a1508c5b3 rebuilded https://testnet4809.swaponline.site https://mainnet4809.swaponline.site

File: src/front/shared/pages/CurrencyWallet/CurrencyWallet.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const isWidgetBuild = config && config.isWidget
 @withRouter
 @CSSModules({ ...styles, ...stylesHere }, { allowMultiple: true })
 class CurrencyWallet extends Component<any, any> {
-  _mounted = false
```
Comment (VitaliyShulik) [inline src/front/shared/pages/CurrencyWallet/CurrencyWallet.tsx:60]: а зачем убрал underscore?

Comment (NotEternal) [reply src/front/shared/pages/CurrencyWallet/CurrencyWallet.tsx:60]: нечего конкретного. Где то с ним используется, где то без. Тут просто решил убрать) подумав что это не важно в компоненте реакта

**Review by VitaliyShulik — APPROVED** [review]

**Pull request merged**